### ISSUE TITLE
Bing Incorrect Blocking [EasyPrivacy]

### ISSUE THREAD

**Issue by superlex** [easylist/easylist#5898]
### List the website(s) you're having issues:

`https://www.bing.com/images/search`

### What happens?

Blank (black) page searching for an image:
https://streamable.com/dtpiez

### List Subscriptions you're using:

- EasyList
- EasyPrivacy
- EasyList Italy

### Your settings

- OS/version: macOS
- Browser/version: Firefox
- Adblock Extension/version: uBlock0 - AdGuard Ext

### Problematic rule: 
`||bing.com/fd/ls/$xmlhttprequest`

### Possible fix:
`@@||bing.com/fd/ls/$xmlhttprequest`

**Comment by THEtomaso** [comment]
Unable to reproduce.
The blocking rule doesn't seem to cause any issues at my end.

**Comment by llacb47** [comment]
Nice try microsoft employee 😅. That rule blocks logging and should not break the page. Try backing up then resetting your adblocker to default settings. Also, I'm unclear - are you using uBlock Origin or Adguard?

**Comment by Khrin** [comment]
@superlex I did the same procedure but I'm also too unable to reproduce the issue, did you encounter it with either Ublock origin and Adguard?

**Comment by superlex** [comment]
@Khrin 
Yep, both Ublock origin and Adguard (one by one).
The problem is 
`bing.com/fd/ls/lsp.aspx`
blocked by EasyPrivacy rule
`||bing.com/fd/ls/$~ping`
that uBlock0 see as
`||bing.com/fd/ls/$xmlhttprequest`
An other user has the same issue.

**Comment by Yuki2718** [comment]
Neither could I - I don't have Mac so tested on Windows and Ubuntu with Firefox + uBO + default lists, no problem with the same procedure.

**Comment by superlex** [comment]
Neither AG Team can reproduce it
https://github.com/AdguardTeam/AdguardFilters/issues/60897
If you prefer close the issue, thanks anyway :)

**Comment by Yuki2718** [comment]
@superlex Which another user, if him/her or you add the exception and reload the page the issue disappears soon and then if removed the exception the issue come back 100% of the time without exception? Could you test that with fresh Firefox profile with only uBO or AdGuard with default settings installed?

**Comment by Yuki2718** [comment]
May worth a new issue, but upon trying to reproduce the original report I happened to get a missed ad. As expected, hard to reproduce - I even don't know how to reproduce the box below the ad which on other Firefox profiles shown in the left side (the SS was taken on a default Firefox profile). If anything, I logged into MS account, searched, logged out and then searched the same. Search language was set to English via Bing config.

![bing](https://user-images.githubusercontent.com/58900598/89700462-2abc5300-d969-11ea-87a1-33c94193e613.png)

![bing2](https://user-images.githubusercontent.com/58900598/89700752-c058e200-d96b-11ea-8a4d-fa55fa9a2bc1.png)

![bing3](https://user-images.githubusercontent.com/58900598/89700758-c3ec6900-d96b-11ea-8798-d9b8cd03efb7.png)

Theoretically it'll be removed by either of
```
bing.com##.ins_exp.tds
bing.com#?#.insml > li :-abp-has(.ins_exp.tds)
```
but I couldn't test - "open this frame in new tab" and it disappeared.

**Comment by llacb47** [comment]
@Yuki2718 

I had reported that a couple months ago, `bing.com##.ins_exp.tds` was working at the time - guess they haven't changed it yet which is good

**Comment by superlex** [comment]
@Yuki2718 
Thanks.
Clean FireFox profile: I cannot reproduce (I tried both uBlock0 and AdGuard).
Old FireFox profile: I can reproduce, id="b_content"  is empty.
I asked the other user to try, too.

**Comment by superlex** [comment]
Neither the other user can reproduce it with a fresh FireFox profile (importing old uBlock0 backup configuration) :)

**Comment by Yuki2718** [comment]
> Neither the other user can reproduce it with a fresh FireFox profile (importing old uBlock0 backup configuration) :)

Then I guess the issue may be related to something corrupted in the profile than the rule itself.

**Comment by superlex** [comment]
@Yuki2718 
Yep, I'll investigate..
